src/collective/cover/static/contentchooser.js

```diff
@@ -50,20 +71,38 @@
         $(windowId + " ul.formTabs").tabs("div.panes > div");
     }
 
+    var parse_url = function(url) {
```

Comment:
need this method to parse the plone default navigation url to get b_start attribute: "http://localhost:8080/Plone/teste/compose?b_start:int=4"

it just breaks the url, convert to int the value, and return a dict with values